base/bucket_gocb.go

```diff
@@ -86,13 +86,14 @@ func GetCouchbaseBucketGoCB(spec BucketSpec) (bucket *CouchbaseBucketGoCB, err e
 	password := ""
 	// Check for client cert (x.509) authentication
 	if spec.Certpath != "" {
+		Infof(KeyAuth, "Attempting cert authentication against bucket %s on %s", MD(spec.BucketName), MD(connString))
```

Comment:
As discussed, it's worth reviewing whether connString can contain credentials when they are prepended to the server in Sync Gateway's config, and ensure they aren't being logged here.  If this logging/approach isn't included in the subsequent PR, we'll need a way to remember to revisit whatever the equivalent logging is.

Reply:
Turns out the connection string is sanitised. If you specify basic auth creds, it'll parse it and remove them from the server URL.